ckanext/resource_type_validation/resources/resource_types.json

```diff
@@ -13,18 +13,26 @@
             "application/sparql-query",
             "application/x-ascii-grid",
             "application/xml",
+            "application/json",
             "application/rdf+xml",
+            "application/vnd.google-earth.kml+xml",
             "text/*"
         ],
         "application/zip": ["application/vnd.google-earth.kmz"],
         "application/CDFV2": [
             "application/vnd.ms-excel",
             "application/msword",
             "application/vnd.ms-powerpoint"
+        ],
+        "application/CDFV2-unknown": [
```

Comment:
Perhaps this could be an `equal_types` entry?

Reply:
I tried replacing that with an `equals_types` entry:
`["application/CDFV2-unknown", "application/CDFV2"]`
but I get the error that says it is unable to determine whether the type is 'application/msword' or 'application/CDFV2-unknown'